StortSpelProjekt/Engine/src/Renderer/Renderer.cpp

```diff
@@ -262,8 +267,22 @@ void Renderer::RenderUpdate(double dt)
 	{
 		ImGuiHandler::GetInstance().NewFrame();
 	}
+	
+	float3 right = m_pScenePrimaryCamera->GetRightVectorFloat3();
+	right.normalize();
+
+	float3 forward = m_pScenePrimaryCamera->GetDirectionFloat3();
+	forward.normalize();
+
+	// TODO: fix camera up vector
```

Comment:
Camera does have up vector function

Reply:
It is fuuuucked. It is always (0, 1, 0) when it should change once the camera changes. I guess the cameras up is world up.